Packs/GLPI/Integrations/GLPI/test/test.txt

```diff
@@ -0,0 +1 @@
+testing
```

Comment:
Please move this to the `test_data` directory.

Reply:
done